prow/flagutil/github.go

```diff
@@ -66,6 +68,10 @@ func (o *GitHubOptions) Validate(dryRun bool) error {
 		}
 	}
 
+	if _, err := url.Parse(o.graphqlEndpoint); err != nil {
```

Comment:
This one doesn't validate that something _is_ passed

Reply:
Empty would only happen if someone explicitly specified it, the default is covered by the stringvar flag default.

I also think this parse would fail on empty.

Reply:
That's the same as in the other place you have written the check, so let's make them identical code.

Reply:
Ohh, I think I know what you are talking about. I basically copied the existing github endpoint check for each file, so the check logic would be the same. I can update the check logic for github endpoint and graphql endpoint in `flagutil/github.go`, but I would prefer to not update any other files.

This goes back to the flagutil consolidation effort we spoke about a week or two ago. 

Sound good?

Reply:
I would just like to see the content of `Validate()` be the same for every place we're asking for this as a flag

Reply:
have you tried rerouting it to .shell maybe it might work

Reply:
another thing. what about the parse code being implemented into something else so it can be passed

Reply:
/uncc

Reply:
/uncc
/cc @Noahhoetger2001